frontend/src/services/list.service.ts

```diff
@@ -29,7 +29,7 @@ const ListService = {
         id: id
       }]
     );
-    const ogData = <IOGData>response.data;
+    const ogData = <IOGData>response.data[0];

```

Comment:
Let's add a unit test for this.

Reply:
At this moment, we have no unit test coverage for front-end. I`ll create a separate deliverable to add UT coverage for FE.